### ISSUE TITLE
Lost Connection to Nextcloud (v26) after Update to 7.9.2

### ISSUE THREAD

**Issue by hartsberger** [ONLYOFFICE/onlyoffice-nextcloud#892]
Hi, 

we have Nextcloud 26.0.7. 

After updating the Onlyoffice app from 7.9.0 to 7.9.2 we get the following error when checking the connection: 

```
sudo -u www-data php occ onlyoffice:documentserver --check

Error connection: Im Dokumentenservice ist ein Fehler aufgetreten: Error while downloading the document file to be converted.
```

The Onlyoffice Docker container shows the following errors: 

```
onlyoffice-document-server_1  |
onlyoffice-document-server_1  | ==> /var/log/onlyoffice/documentserver/converter/out.log <==
onlyoffice-document-server_1  | [2023-10-16T08:31:18.933] [ERROR] [localhost] [conv_check_1241478269_docx] [userId] nodeJS - error downloadFile:url=https://xxx/index.php/apps/onlyoffice/empty?doc=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY3Rpb24iOiJlbXB0eSJ9.yl6O62n5kJaH5Soqop5qoDxQyeooBoIyXN-L2_Snn38;attempt=1;code:undefined;connect:undefined Error: Error response: statusCode:404; headers:{"date":"Mon, 16 Oct 2023 06:30:57 GMT","server":"Apache/2.4.52 (Ubuntu)","referrer-policy":"no-referrer","x-content-type-options":"nosniff","x-frame-options":"SAMEORIGIN","x-permitted-cross-domain-policies":"none","x-robots-tag":"noindex, nofollow","x-xss-protection":"1; mode=block","set-cookie":["ocdlkbru62ei=b84mvgl8oqi7cuth6f7m72; path=/; secure; HttpOnly; SameSite=Lax","oc_sessionPassphrase=mYvj9xvSGHkOtfk28KhmxwB6klnCumXmAuVpsFwyTVlBGMtL6ylXpqijoPwEai5O83eUsLc8A2Nxo1g7DYZuKDkyNBNE2z7%2F3stp2CQen74HeI85lI4hdOxkA; path=/; secure; HttpOnly; SameSite=Lax","ocdlkbru62ei=df81rfpqdq3ut6sqo5voquj00t; path=/; secure; HttpOnly; SameSite=Lax","ocdlkbru62ei=df81rfpqdq3ut6sqo5voquj00t; path=/; secure; HttpOnly; SameSite=Lax","__Host-nc_sameSiteCookielax=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=lax","__Host-nc_sameSiteCookiestrict=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=strict","ocdlkbru62ei=df81rfpqdq3ut6sqo5voquj00t; path=/; secure; HttpOnly; SameSite=Lax"],"expires":"Thu, 19 Nov 1981 08:52:00 GMT","cache-control":"no-cache, no-store, must-revalidate","pragma":"no-cache","content-security-policy":"default-src 'none';base-uri 'none';manifest-src 'self';frame-ancestors 'none'","x-request-id":"oB3LvUEZB4zjcFLKZOO7","feature-policy":"autoplay 'none';camera 'none';fullscreen 'none';geolocation 'none';microphone 'none';payment 'none'","content-length":"34","strict-transport-security":"max-age=15768000;includeSubdomains","keep-alive":"timeout=5, max=100","connection":"Keep-Alive","content-type":"application/json; charset=utf-8"};
onlyoffice-document-server_1  |     at Request.fResponse (/snapshot/server/build/server/Common/sources/utils.js)
onlyoffice-document-server_1  |     at Request.emit (events.js:400:28)
onlyoffice-document-server_1  |     at Request.onRequestResponse (/snapshot/server/build/server/Common/node_modules/request/request.js:1059:10)
onlyoffice-document-server_1  |     at ClientRequest.emit (events.js:400:28)
onlyoffice-document-server_1  |     at HTTPParser.parserOnIncomingClient (_http_client.js:647:27)
onlyoffice-document-server_1  |     at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
onlyoffice-document-server_1  |     at HTTPParser.execute (<anonymous>)
onlyoffice-document-server_1  |     at TLSSocket.socketOnData (_http_client.js:515:22)
onlyoffice-document-server_1  |     at TLSSocket.emit (events.js:400:28)
onlyoffice-document-server_1  |     at addChunk (internal/streams/readable.js:293:12)
onlyoffice-document-server_1  |     at readableAddChunk (internal/streams/readable.js:267:9)
onlyoffice-document-server_1  |     at TLSSocket.Readable.push (internal/streams/readable.js:206:10)
onlyoffice-document-server_1  |     at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)
onlyoffice-document-server_1  | [2023-10-16T08:31:19.987] [ERROR] [localhost] [conv_check_1241478269_docx] [userId] nodeJS - error downloadFile:url=https://xxx/index.php/apps/onlyoffice/empty?doc=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY3Rpb24iOiJlbXB0eSJ9.yl6O62n5kJaH5Soqop5qoDxQyeooBoIyXN-L2_Snn38;attempt=2;code:undefined;connect:undefined Error: Error response: statusCode:404; headers:{"date":"Mon, 16 Oct 2023 06:30:58 GMT","server":"Apache/2.4.52 (Ubuntu)","referrer-policy":"no-referrer","x-content-type-options":"nosniff","x-frame-options":"SAMEORIGIN","x-permitted-cross-domain-policies":"none","x-robots-tag":"noindex, nofollow","x-xss-protection":"1; mode=block","set-cookie":["ocdlkbru62ei=fm8ogmhmhjm57qpdvreh; path=/; secure; HttpOnly; SameSite=Lax","oc_sessionPassphrase=mOLpxr2BfGZGcdIjv0gPyfwmfsyI3kdbzCd9OfF07QUgvxMOuhyiDGYG%2BekpaJPG%2F1FnZvMMpfVR6LBh3zy7w327%2BFnjaGNv7uYRqi2k3uKY6pVBIsVAvK5D; path=/; secure; HttpOnly; SameSite=Lax","ocdlkbru62ei=2fi2kaimdlerjdllfuaa59shfs; path=/; secure; HttpOnly; SameSite=Lax","ocdlkbru62ei=2fi2kaimdlerjdllfuaa59shfs; path=/; secure; HttpOnly; SameSite=Lax","__Host-nc_sameSiteCookielax=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=lax","__Host-nc_sameSiteCookiestrict=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=strict","ocdlkbru62ei=2fi2kaimdlerjdllfuaa59shfs; path=/; secure; HttpOnly; SameSite=Lax"],"expires":"Thu, 19 Nov 1981 08:52:00 GMT","cache-control":"no-cache, no-store, must-revalidate","pragma":"no-cache","content-security-policy":"default-src 'none';base-uri 'none';manifest-src 'self';frame-ancestors 'none'","x-request-id":"umPeJyEOczDMBkB1THM9","feature-policy":"autoplay 'none';camera 'none';fullscreen 'none';geolocation 'none';microphone 'none';payment 'none'","content-length":"34","strict-transport-security":"max-age=15768000;includeSubdomains","keep-alive":"timeout=5, max=100","connection":"Keep-Alive","content-type":"application/json; charset=utf-8"};
onlyoffice-document-server_1  |     at Request.fResponse (/snapshot/server/build/server/Common/sources/utils.js)
onlyoffice-document-server_1  |     at Request.emit (events.js:400:28)
onlyoffice-document-server_1  |     at Request.onRequestResponse (/snapshot/server/build/server/Common/node_modules/request/request.js:1059:10)
onlyoffice-document-server_1  |     at ClientRequest.emit (events.js:400:28)
onlyoffice-document-server_1  |     at HTTPParser.parserOnIncomingClient (_http_client.js:647:27)
onlyoffice-document-server_1  |     at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
onlyoffice-document-server_1  |     at HTTPParser.execute (<anonymous>)
onlyoffice-document-server_1  |     at TLSSocket.socketOnData (_http_client.js:515:22)
onlyoffice-document-server_1  |     at TLSSocket.emit (events.js:400:28)
onlyoffice-document-server_1  |     at addChunk (internal/streams/readable.js:293:12)
onlyoffice-document-server_1  |     at readableAddChunk (internal/streams/readable.js:267:9)
onlyoffice-document-server_1  |     at TLSSocket.Readable.push (internal/streams/readable.js:206:10)
onlyoffice-document-server_1  |     at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)
onlyoffice-document-server_1  | [2023-10-16T08:31:21.050] [ERROR] [localhost] [conv_check_1241478269_docx] [userId] nodeJS - error downloadFile:url=https://xxx/index.php/apps/onlyoffice/empty?doc=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY3Rpb24iOiJlbXB0eSJ9.yl6O62n5kJaH5Soqop5qoDxQyeooBoIyXN-L2_Snn38;attempt=3;code:undefined;connect:undefined Error: Error response: statusCode:404; headers:{"date":"Mon, 16 Oct 2023 06:30:59 GMT","server":"Apache/2.4.52 (Ubuntu)","referrer-policy":"no-referrer","x-content-type-options":"nosniff","x-frame-options":"SAMEORIGIN","x-permitted-cross-domain-policies":"none","x-robots-tag":"noindex, nofollow","x-xss-protection":"1; mode=block","set-cookie":["ocdlkbru62ei=a8ki1cp9hdie05pou1k8; path=/; secure; HttpOnly; SameSite=Lax","oc_sessionPassphrase=BXlSjY3ffbczx2cLEi8R6Y6fYXtgHPVUnGBb%2FZq4oaJytSB4oMt9WtNLkRPQxFPI6nz4LGakRXfBzpWYdXCvM4HN8IUaIHO3IWaC%2F1nlSP8UGfyl3kWbGxS; path=/; secure; HttpOnly; SameSite=Lax","ocdlkbru62ei=cd3egd6qf9ok7r8s8cg0i8oall; path=/; secure; HttpOnly; SameSite=Lax","ocdlkbru62ei=cd3egd6qf9ok7r8s8cg0i8oall; path=/; secure; HttpOnly; SameSite=Lax","__Host-nc_sameSiteCookielax=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=lax","__Host-nc_sameSiteCookiestrict=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=strict","ocdlkbru62ei=cd3egd6qf9ok7r8s8cg0i8oall; path=/; secure; HttpOnly; SameSite=Lax"],"expires":"Thu, 19 Nov 1981 08:52:00 GMT","cache-control":"no-cache, no-store, must-revalidate","pragma":"no-cache","content-security-policy":"default-src 'none';base-uri 'none';manifest-src 'self';frame-ancestors 'none'","x-request-id":"1rfVrTfEEH6hF5ayQtnd","feature-policy":"autoplay 'none';camera 'none';fullscreen 'none';geolocation 'none';microphone 'none';payment 'none'","content-length":"34","strict-transport-security":"max-age=15768000;includeSubdomains","keep-alive":"timeout=5, max=100","connection":"Keep-Alive","content-type":"application/json; charset=utf-8"};
onlyoffice-document-server_1  |     at Request.fResponse (/snapshot/server/build/server/Common/sources/utils.js)
onlyoffice-document-server_1  |     at Request.emit (events.js:400:28)
onlyoffice-document-server_1  |     at Request.onRequestResponse (/snapshot/server/build/server/Common/node_modules/request/request.js:1059:10)
onlyoffice-document-server_1  |     at ClientRequest.emit (events.js:400:28)
onlyoffice-document-server_1  |     at HTTPParser.parserOnIncomingClient (_http_client.js:647:27)
onlyoffice-document-server_1  |     at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
onlyoffice-document-server_1  |     at HTTPParser.execute (<anonymous>)
onlyoffice-document-server_1  |     at TLSSocket.socketOnData (_http_client.js:515:22)
onlyoffice-document-server_1  |     at TLSSocket.emit (events.js:400:28)
onlyoffice-document-server_1  |     at addChunk (internal/streams/readable.js:293:12)
onlyoffice-document-server_1  |     at readableAddChunk (internal/streams/readable.js:267:9)
onlyoffice-document-server_1  |     at TLSSocket.Readable.push (internal/streams/readable.js:206:10)
onlyoffice-document-server_1  |     at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)
```

Onlyoffice works fine with another Nextcloud instance (v23) that is running an older version of the Onlyoffice app (7.4.8). 

I reverted back to 7.9.0 and Onlyoffice works fine again. 

There must be something wrong with the new version.

**Comment by KornKalle** [comment]
Can confirm this. Nextcloud 26.0.7 and onlyoffice 7.9.2 connecting to OnlyOffice enterprise Server

**Comment by hartsberger** [comment]
Problem solved with Onlyoffice App version 7.9.4 (and Nextcloud update to 26.0.8)

```
cd /var/www/nextcloud/ && sudo -u www-data php occ onlyoffice:documentserver --check
Document server https://onlyoffice.stadtwerke-dachau.de/ version 7.4.1.36 is successfully connected
```

Thanks.

**Comment by SergeyKorneyev** [comment]
Hi @vbelloir 
Could you create a separate issue for this?
In any case, the first thing you could try is to put the cloud.domain.fr and office.domain.fr addresses in the corresponding fields in the Advanced server settings section of the connector's settings.
